meta-cube/classes/c3-app-container.bbclass

```diff
@@ -9,10 +9,6 @@ LIC_FILES_CHKSUM ?= "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecd
 IMAGE_FSTYPES ?= "tar.bz2"
 IMAGE_FSTYPES_remove = "live"
 
-TARGETNAME ?= "c3-app-container"
```

Comment:
Why wouldn't we still set TARGETNAME in the bbclass ?

Reply:
I tried setting `TARGETNAME = "<some_name>"` but the generated image's name is the same as the recipe name. hence `TARGETNAME ?= "c3-app-container"` seemed like a no-op to me. In short, it doesn't seem like the build system makes use of the variable. I might be wrong.

Reply:
It is used in hosts.bbclass, it is about naming inside the running container, not the name of the container image name. If it isn't working, that's a bug and we should fix the bug, but the variable should be left as set by the class/.inc

Another thing that would need to be changed if we move to a bbclass would be the Documentation of the layer. It currently describes the .inc mechanism, and would need to be updated.

Reply:
ok thank you @zeddii. I will make a the requested changes and send a v2